examples/configs/evaluation/lm_harness_hf_config.yaml

```diff
@@ -1,3 +1,5 @@
+name: "lm-buddy-lm-harness"
```

Comment:
I've decided to add a `name` field as a top-level parameter on on all `LMBuddyJobConfig`s. This is so we always have a consistent name for referring to the job run and any assets produced by it (e.g., the artifact names).

This is a pretty common pattern, if you take a look at MosaicML and how they handle their `RunConfig` object.